doc/user-guide/user-guide.rst

```diff
@@ -0,0 +1,72 @@
+..
+  This is a template for the user-guide documentation that will accompany each package.
+  This template is provided to ensure that the documentation remains similar in look, feel, and contents to users.
+  The headings below are expected to be present for all packages, but for many packages, additional fields will be required.
+
+  ** All text in square brackets [] must be re-populated accordingly **
+
+  See https://developer.lsst.io/restructuredtext/style.html
+  for a guide to reStructuredText writing.
+
+  Use the following syntax for sections:
+
+  Sections
+  ========
+
+  and
+
+  Subsections
+  -----------
+
+  and
+
+  Subsubsections
+  ^^^^^^^^^^^^^^
+
+  To add images, add the image file (png, svg or jpeg preferred) to the
+  images/ directory. The reST syntax for adding the image is
+
+  .. figure:: /images/filename.ext
+   :name: fig-label
+
+   Caption text.
+
+  Feel free to delete this instructional comment.
+
+.. Fill out data so contacts section below is auto-populated
+.. add name and email between the *'s below e.g. *Marie Smith <msmith@lsst.org>*
+
+.. _User_Guide:
+
+###########################
+IntegrationTests User Guide
+###########################
+
+.. Update links and labels below
+.. image:: https://img.shields.io/badge/GitHub-ts_athexapod-green.svg
```

Comment:
These all point to ATHexpod. Should be changed to point to this package.

Reply:
fixed